index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<div class ="header">
+<style>
+table, th, td {
+    border: 1px solid black;
+}
+</style>
+</div class="header">
```

Comment:
Attributes are usually on the opening tag and not the closing tag
This:
`<div class="header"></div>`
is more conventional than this:
`<div></div class="header">`

Reply:
Usually for headers we will use `<hX>` where X is a number 1-6 based on header value with 1 as the primary for things likes titles and 6 the smallest.